src/main/java/fi/aalto/cs/apluscourses/model/Course.java

```diff
@@ -52,6 +56,12 @@ public static Course createEmptyCourse() {
     return new Course("", new ArrayList<>(), new HashMap<>());
```

Comment:
Nit:
I understand this but suggest to consider having a separate `CourseTestHelper` class to work with

Reply:
This method became actually obsolete in this PR. I'll remove it altogether. It wasn't originally intended (by Nikolas) for testing.